From 0da948aa667065c1940c517256ee39a2494b142d Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 31 Jan 2013 18:04:45 +0800
Subject: [PATCH 413/486] ENGR00242437 HDMI Single: UI reboot if hot plug the
 HDMI cable

Don't report HDMI hotplug event when HDMI is primary display

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 mx6/hwcomposer/hwc_vsync.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mx6/hwcomposer/hwc_vsync.cpp b/mx6/hwcomposer/hwc_vsync.cpp
index e4fade9..c83cd7d 100644
--- a/mx6/hwcomposer/hwc_vsync.cpp
+++ b/mx6/hwcomposer/hwc_vsync.cpp
@@ -125,7 +125,8 @@ bool VSyncThread::threadLoop()
     if(vsync) {
         handleVsyncUevent(uevent_desc, len);
     }
-    else if(hdmi) {
+    else if (hdmi &&
+            mCtx->mDispInfo[HWC_DISPLAY_PRIMARY].type != HWC_DISPLAY_HDMI) {
         handleHdmiUevent(uevent_desc, len);
     }
 
-- 
1.8.0

